### ISSUE TITLE
`no-invalid-regexp` does not look inside function calls

### ISSUE THREAD

**Issue by 0f-0b** [denoland/deno_lint#1269]
### Lint Name

`no-invalid-regexp`

### Code Snippet

```js
((_) => [
  /+/,
  RegExp("+"),
  new RegExp("+"),
])([
  /+/,
  RegExp("+"),
  new RegExp("+"),
]);
```

### Expected Result

One warning for each regex.

### Actual Result

No warning.

### Version

```shellsession
$ dlint --version
dlint 0.58.3
```

**Comment by Colt45s** [comment]
May I do it?
